app/uk/gov/hmrc/eoricommoncomponent/frontend/views/ServiceName.scala

```diff
@@ -42,6 +42,6 @@ object ServiceName {
     shortName(service)
```

Comment:
You need to update this class to get the long and short names from the Service class instead of messages file.